README.md

```diff
@@ -59,6 +58,12 @@ changes may be written to a config file in a platform-specific location:
   Alternatively, a development build can be performed by running `go install` in
   a locally checked-out repository.
 
+  If you want to easily access dcrctl via `dcrctl` handle on command-line, since
```

Comment:
This doesn't read correctly to me because `dcrctl` is definitely not an alias or handle.  Those are different things entirely.

Also, this is linux-specific without calling it out.

Maybe something that works for all OSes like:

> If you want to easily access dcrctl from the command-line without having to type the full path to the binary every time, ensure the `bin` directory rooted at the path reported by `go env GOPATH` is added to your system path.
>
> 

Then link to OS-specific resources.  I'd suggest one for macOS, Windows, and Linux to match the ones listed previously for configuration files.

Reply:
Applied, and added links to OS-specific tutorials on how to add binary to PATH.